src/Spout/Reader/Helper/XLSX/SharedStringsHelper.php

```diff
@@ -34,6 +34,9 @@ class SharedStringsHelper
     /** Value to use to escape the line feed character ("\n") */
     const ESCAPED_LINE_FEED_CHARACTER = '_x000A_';
 
+    /** This will increase your memory usage but can improve your execution time */
+    static public $KEEP_ALL_STRINGS_IN_MEMORY = false;
```

Comment:
I'm not a big fan of exposing this global method to support messed up files.
One of the goal of Spout is to read any file without going out of memory. This is something other libraries can't do and that I'd like to preserve. Adding an option to disable caching goes against this philosophy as it will make it possible to run out of memory (what if you had 500K strings instead of 130K?). In this case, I would argue that it's better to have a slow program than a one that crashes.

Does it make sense?